integration-tests/simple-webapp/src/main/java/co/elastic/webapp/ExecuteCmdServlet.java

```diff
@@ -0,0 +1,75 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2019 Elastic and contributors
+ * %%
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * #L%
+ */
+package co.elastic.webapp;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+public class ExecuteCmdServlet extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException {
+        String[] cmd = new String[]{getJavaBinaryPath(), "-version"};
+
+        int returnValue;
+        try {
+            Process process = Runtime.getRuntime().exec(cmd);
```

Comment:
Is it on purpose the integration test only tests the process created with the Runtime class?
Since the ProcessBuilder has a own instrumentation class I would expect it to also have an integration test.

Reply:
You are right, `Runtime.getRuntime().exec(...)` delegates to `ProcessBuilder`, but that's an implementation detail.

As far as I know creating instances of `Process` is always done through `ProcessBuilder.start`, thus in this case I'd say that it's pretty safe.

However, supporting java9 `ProcessHandler` for example would require to have a different code here to start and wait for process termination.